src/test/java/edu/ucsb/courses/documents/statistics/SingleCourseSearchTests.java

```diff
@@ -0,0 +1,68 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
+import net.codebox.javabeantester.JavaBeanTester;
+import org.aspectj.apache.bcel.classfile.Module;
+import org.junit.jupiter.api.BeforeEach;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
```

Comment:
Unused import, we should remove.

Reply:
We removed the unused import

Reply:
We used the first three but did not use the last one so we removed that one.